src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts

```diff
@@ -1,32 +1,36 @@
-import {AfterViewInit, Component, OnDestroy, OnInit} from '@angular/core';
-import {FormBuilder, FormGroup, Validators} from '@angular/forms';
-import {ActivatedRoute} from '@angular/router';
-import {select, Store} from '@ngrx/store';
-import {Subscription} from 'rxjs';
-import {ACTION, HearingLinkMessages} from '../../../models/hearings.enum';
-import {ServiceLinkedCasesModel} from '../../../models/linkHearings.model';
-import {HearingsService} from '../../../services/hearings.service';
+import { AfterViewInit, Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { ACTION, HearingLinkMessages } from '../../../models/hearings.enum';
+import { ServiceLinkedCasesModel } from '../../../models/linkHearings.model';
+import { LovRefDataModel } from '../../../models/lovRefData.model';
+import { HearingsService } from '../../../services/hearings.service';
 import * as fromHearingStore from '../../../store';
-import {RequestHearingPageFlow} from '../request-hearing.page.flow';
+import { RequestHearingPageFlow } from '../request-hearing.page.flow';
 
 @Component({
   selector: 'exui-hearing-link',
   templateUrl: './hearing-link.component.html'
 })
 export class HearingLinkComponent extends RequestHearingPageFlow implements OnInit, AfterViewInit, OnDestroy {
   public caseId: string;
-  public linkedCases: ServiceLinkedCasesModel[];
+  public linkedCases: ServiceLinkedCasesModel[] = [];
+  public caseLinkingReasonCodes: LovRefDataModel[] = [];
```

Comment:
you've made all these things public, but do they need to be public?

Reply:
This will be done as part of the PR https://github.com/hmcts/rpx-xui-webapp/pull/2630 which you already commented on and merged later into this.